src/amuse/test/suite/compile_tests/test_stopping_conditions.py

```diff
@@ -573,7 +573,7 @@ def get_mpidir(cls):
         
     @classmethod
     def get_codestring(cls):
-        return CodeStringF
+        return codestringF
```

Comment:
so is pytest still testing this class (it shouldn't)?

Reply:
yes, this came up as an error.

Reply:
the change is ok, but it shouldn't test the class